### PR TITLE
To do rosacli testing with the code under the rosa repository

### PR DESCRIPTION
**Pull request by yasun1**
Relied on https://github.com/openshift/release/pull/50520

[OCM-6310](https://issues.redhat.com/browse/OCM-6310)

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hcp-e2e-f3

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-e2e-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hcp-e2e-f3

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml [hunk 1]
```diff
@@ -47,17 +49,21 @@ tests:
       ENABLE_AUTOSCALING: "true"
       MAX_REPLICAS: "6"
       MIN_REPLICAS: "3"
+      TEST_LABEL_FILTERS: day2
```
Comment (xueli181114) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:52]: day1-post and non-exclude cases should be included please. Not only day2

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hcp-security-group-e2e-f7

**Comment by yasun1** [comment]
@liangxia Please help review. Thanks

**Pull request merged**